asistente_ladm_col/config/db_mapping_config.py

```diff
@@ -427,6 +427,20 @@ class DBMappingConfig:
                     'Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_InteresadoContacto.Telefono2': 'LC_PARTY_CONTACT_T_TELEPHONE_NUMBER_2_F',
                     'Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.lc_interesado_interesadocontacto.lc_interesado..Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_Interesado': 'LC_PARTY_CONTACT_T_LC_PARTY_F'
                 }},
+            "LADM_COL.LADM_Nucleo.COL_AreaValor": {QueryNames.VARIABLE_NAME: "COL_VALUE_AREA_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_TipologiaConstruccion": {QueryNames.VARIABLE_NAME: "LC_TIPOLOGY_BUILDING_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_CalificacionNoConvencional": {QueryNames.VARIABLE_NAME: "LC_NON_CONVENTIONAL_QUALIFICATION_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_CalificacionConvencional": {QueryNames.VARIABLE_NAME: "LC_CONVENTIONAL_QUALIFICATION_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_GrupoCalificacion": {QueryNames.VARIABLE_NAME: "LC_GROUP_QUALIFICATION_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_ObjetoConstruccion": {QueryNames.VARIABLE_NAME: "LC_BUILDING_OBJECT_T", QueryNames.FIELDS_DICT: {}},
```

Comment:
Please, rename LC_SURVEY_CADASTRAL_ADITIONAL_DATA_T by LC_SURVEY_CADASTRAL_ADDITIONAL_DATA_T

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/577/commits/4888b2f347196a8dc65bad715dc7f8c71a5731dd